### ISSUE TITLE
Missing break statements after VALUE_FALSE in deserialize method

### ISSUE THREAD

**Issue by lukaszstempin** [graphql-java-generator/graphql-maven-plugin-project#200]
It looks like there is a missing 'break' after VALUE_FALSE. 
Without it, the boolean value always tries to parse as a number, which of course will never succeed.

```switch (p.currentToken()) {
case VALUE_FALSE:
case VALUE_TRUE:
   value = new BooleanValue(p.getBooleanValue());
case VALUE_NUMBER_FLOAT:
   value = new FloatValue(p.getDecimalValue());
   break;
case VALUE_NUMBER_INT:
   value = new IntValue(p.getBigIntegerValue());
   break;
case VALUE_STRING:
   value = new StringValue(p.getText());
   break;
case VALUE_NULL:
   value = null;
   break;
default:
   throw new JsonParseException(p, "Non managed JSON token: " + p.currentToken());
			}
```

My suggestion:
```case VALUE_FALSE:
case VALUE_TRUE:
   value = new BooleanValue(p.getBooleanValue());
   break;
```

**Comment by lukaszstempin** [comment]
I wanted to raise a PR for this but looks like I am not able to do so.

> Please make sure you have the correct access rights and the repository exists.

**Comment by etienne-sf** [comment]
Hello,

Thanks for the hint. 
I think everybody may submit PR. To do this:
* clone the project in GitHub, into your repositories. 
* do the change. 
* submit the PR

Étienne

**Comment by etienne-sf** [comment]
And an additional note: I'd be happy to validate this PR. 

What would be even greater, is if you add Junit test that would be ko before the correction, and become ok after it.

**Comment by etienne-sf** [comment]
Question : do you plan to do this PR ?
Otherwise I commit the correction you pointed out.

**Comment by lukaszstempin** [comment]
Hey @etienne-sf unfortunately I have no access to raise a PR. Sorry for that : /

**Comment by etienne-sf** [comment]
Ok, no problem 

I'll commit this correction. 
:)

**Comment by etienne-sf** [comment]
Solved in the 2.3.1 release.
Along with a bug that would prevent to use alias on field which type is a custom scalar.
